bin/git-psykorebase

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+PROGRAM=$0
+PRIMARY_BRANCH=""
+SECONDARY_BRANCH=""
+FF="--ff"
+CONTINUE="no"
+
+function current_branch() {
+    git rev-parse --abbrev-ref HEAD
+}
+
+function usage()
+{
+    echo "USAGE: ${PROGRAM} PRIMARY_BRANCH [SECONDARY_BRANCH]"
+    echo "USAGE: ${PROGRAM} --continue"
```

Comment:
I would argue that people understand what  ${0} is, so you can drop the variable. But you make the call.